File: example/src/main/java/com/github/paolorotolo/appintroexample/ui/mainTabs/DefaultLayout2Intro.java
package com.github.paolorotolo.appintroexample.ui.mainTabs;

import android.app.Fragment;
import android.content.Intent;
import android.os.Bundle;
import android.support.annotation.Nullable;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.Button;

import com.github.paolorotolo.appintroexample.R;
import com.github.paolorotolo.appintroexample.ui.mainTabs.intro.DefaultIntro2;

/**
 * Created by andrew on 11/17/16.
 */

public class DefaultLayout2Intro extends Fragment {

    @SuppressWarnings("ConstantConditions")
    @Override
    public void onActivityCreated(@Nullable Bundle savedInstanceState) {
        super.onActivityCreated(savedInstanceState);

        Button demo = (Button) getView().findViewById(R.id.intro2);
        demo.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                startActivity(new Intent(getActivity().getBaseContext(), DefaultIntro2.class));
            }
        });
    }

    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        return inflater.inflate(R.layout.tab_default_intro2, container, false);
    }
}

File: example/src/main/java/com/github/appintro/example/ui/mainTabs/DefaultLayout2Intro.kt
package com.github.appintro.example.ui.mainTabs

import android.content.Intent
import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import com.github.appintro.appintroexample.R
import com.github.appintro.example.ui.mainTabs.intro.DefaultIntro2
import kotlinx.android.synthetic.main.tab_default_intro2.*

class DefaultLayout2Intro : Fragment() {
    override fun onActivityCreated(savedInstanceState: Bundle?) {
        super.onActivityCreated(savedInstanceState)
        intro2.setOnClickListener {
            startActivity(Intent(context, DefaultIntro2::class.java))
        }
    }

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? =
            inflater.inflate(R.layout.tab_default_intro2, container, false)
}